mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+    </dataUrls>
+    <preprocessing>
+    </preprocessing>
+    <tasks>
+      - climatologyMapSST
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      SSS from NASA Aquarius satellite
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      Level 3 Aquarius sea surface salinity (SSS) data products have a temporal
+      resolutions of daily, 8 day, monthly, 3 months, and annual.  Monthly and
+      seasonal climatology
+      products from Aqaurius are also available. The Aquarius instrument
+      provides global coverage every 7 days. L3 products are gridded at 1
+      degree spatial resolution.
+    </description>
+    <source>
+      [NASA Aquarius FTP server]
+      (ftp://podaac-ftp.jpl.nasa.gov/allData/aquarius/L3/mapped/V4/7day_running/SCI/)
+    </source>
+    <releasePolicy>
+      NASA data are not copyrighted; however, when you publish our data or
+      results derived therefrom, we request that you include an acknowledgment
+      within the text of the publication and reference list.
+      [Data Citation and Acknowledgements]
+      (https://podaac.jpl.nasa.gov/CitingPODAAC)
+    </releasePolicy>
+    <references>
+      [Lagerloef et al. (2015)](ftp://podaac.jpl.nasa.gov/SalinityDensity/aquarius/docs/v4/AQ-014-PS-0016_AquariusSalinityDataValidationAnalysis_DatasetVersion4.0and3.0.pdf)
+    </references>
+    <bibtex>
+    </bibtex>
+    <dataUrls>
+      (missing)
+    </dataUrls>
+    <preprocessing>
+      (missing)
+    </preprocessing>
+    <tasks>
+      - climatologyMapSSS
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      AVISO Absolute Dynamic Topography
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      [NASA JPL AVISO website](https://podaac.jpl.nasa.gov/dataset/AVISO_L4_DYN_TOPO_1DEG_1MO)
+      This dataset contains absolute dynamic topography (similar to sea level
+      but with respect to the geoid) binned and averaged monthly on 1 degree
+      grids. The coverage is from October 1992 to December 2010. These data
+      were provided by AVISO (French space agency data provider) to support the
+      CMIP5 (Coupled Model Intercomparison Project Phase 5) under the World
+      Climate Research Program (WCRP) and was first made available via the JPL
+      Earth System Grid. The dynamic topography are derived from sea surface
+      height measured by several satellites including Envisat, TOPEX/Poseidon,
+      Jason-1 and OSTM/Jason-2, and referenced to the geoid. Along with this
+      dataset, two additional ancillary data files are included in the same
+      directory which contain the number of observations and standard error
+      co-located on the same 1 degree grids.
+    </description>
+    <source>
+      - [AVISO+ website](https://www.aviso.altimetry.fr/en/data/products/sea-surface-height-products/global/madt-h-uv.html)
```

Comment:
I would remove this link, since we got the data from the NASA JPL site. Seems confusing to me.

Reply:
Okay, fine with me.  If you make any other changes to this branch, feel free to remove this link at the same time.